### DISCUSSION TITLE
Dark/Light Mode Switch

### DISCUSSION THREAD

**Discussion by eben80** [dl9rdz/rdz_ttgo_sonde#242] [Developers corner]
Would anyone be interested in a toggle with persistence for setting dark/light mode?
I like dark mode, but would like to be able to switch it briefly sometimes.
Something like this

https://user-images.githubusercontent.com/2545460/139886935-3cb34052-3287-4a35-8b39-9049ad662eac.mp4

:

**Comment by LukePrior** [comment]
Yeah I think that could work well if we continued to set the default theme using system settings but allow the user to switch user the button and that preference is saved.

I'm not sure on the design of that specific icon shown in the video but I think the concept is good.

**Comment by fruijs** [comment]
The dark theme is great but the dark map is almost unreadable while the other menu's are good to read. I do mis a button on the livemap to switch to light and dark mapmaterial. Maybe the excisting mapbutton could be used to toggle between light,dark and satellite?

**Reply by eben80** [reply]
I created a PR for this. I also like to be able to choose it independently.

**Comment by eben80** [comment]
Hi everyone. I did some work on the simulator and used the CSS inverse filter to make things more simple and compact.
See here, all pages now included to reflect all menus:  https://ebski.co/ttgo/index.html#

What are your thoughts? Is the inverse filter sufficient or do we really need full CSS styling?

**Reply by RA3TOX** [reply]
Good option! Greetings.
